common-bsp/recipes-kernel/device-tree-overlays/device-tree-overlays_git.bb

```diff
@@ -0,0 +1,33 @@
+DESCRIPTION = "Device Tree Overlays for bb.org boards"
+
+LICENSE = "MIT"
```

Comment:
GPLv2 would be your best bet, the MIT license is for the recipes themselve, not the contents of the resulting packages